### ISSUE TITLE
ttItemRef.lua:2306: attempt to call field 'IsCurrencyContainer' (a nil value)

### ISSUE THREAD

**Issue by NayooZ** [frozn/TipTac#92]
Happens when looting Badge of Justice (no longer an item - moved to currency). Also happened when accepting daily heroic dungeon quest.

```
2x TipTacItemRef\ttItemRef.lua:2306: attempt to call field 'IsCurrencyContainer' (a nil value)
[string "@TipTacItemRef\ttItemRef.lua"]:2306: in function `?'
[string "@TipTacItemRef\ttItemRef.lua"]:487: in function `SetHyperlink_Hook'
[string "@TipTacItemRef\ttItemRef.lua"]:1467: in function <TipTacItemRef\ttItemRef.lua:1466>
[string "=[C]"]: ?
[string "=[C]"]: in function `SetHyperlink'
[string "@Prat-3.0-3.9.1-6-f65d85e\modules\HoverTips.lua"]:230: in function `?'
[string "@Bartender4\libs\AceHook-3.0\AceHook-3.0-9.lua"]:90: in function <...eBartender4\libs\AceHook-3.0\AceHook-3.0.lua:87>

Locals:
self = GameTooltip {
 BottomLeftCorner = <unnamed> {
 }
 NRCSeparator = <unnamed> {
 }
 ttNumLines = 0
 ttWorkaroundForFirstMouseoverOwner = BuffButton13 {
 }
 ttWorkaroundForFirstMouseoverStatus = 3
 GetBackdropColor = <function> defined @SharedXML\Backdrop.lua:390
 SetHyperlink = <function> defined =[C]:-1
 SetTradeSkillItem = <function> defined =[C]:-1
 _owner = QuestieFrame1176 {
 }
 ttBackdropBorderColorApplied = false
 questOrder = <table> {
 }
 ttSetBackdropBorderColorLocked = true
 ClearBackdrop = <function> defined @SharedXML\Backdrop.lua:282
 SetPetAction = <function> defined =[C]:-1
 NRCSeparator2 = <unnamed> {
 }
 SetLootItem = <function> defined =[C]:-1
 ttDefaultAnchored = false
 ttGradient = <unnamed> {
 }
 numMoneyFrames = 1
 SetQuestCurrency = <function> defined =[C]:-1
 OnBackdropLoaded = <function> defined @SharedXML\Backdrop.lua:152
 SetCurrencyTokenByID = <function> defined =[C]:-1
 SetupTextureCoordinates = <function> defined @SharedXML\Backdrop.lua:214
 TopRightCorner = <unnamed> {
 }
 ttLastUpdate = 0
 ttIcon = <unnamed> {
 }
 GetEdgeSize = <function> defined @SharedXML\Backdrop.lua:188
 NineSlice = <unnamed> {
 }
 SetInventoryItem = <function> defined =[C]:-1
 TopEdge = <unnamed> {
 }
 ttCount = <unnamed> {
 }
 SetLootRollItem = <function> defined =[C]:-1
 ttWorkaroundForFirstMouseoverID = "54646"
 SetBagItem = <function> defined =[C]:-1
 BottomRightCorner = <unnamed> {
 }
 SetBackdropColor = <function> defined @TipTac\ttCore.lua:1638
 LeftEdge = <unnamed> {
 }
 SetAuctionItem = <function> defined =[C]:-1
 ttSetIconTextureAndText = <function> defined @TipTacItemRef\ttItemRef.lua:171
 SetSpellBookItem = <function> defined =[C]:-1
 npcOrder = <table> {
 }
 0 = <userdata>
 SetCurrencyToken = <function> defined =[C]:-1
 SetGuildBankItem = <function> defined =[C]:-1
 hasMoney = 1
 TopLeftCorner = <unnamed> {
 }
 SetUnit = <function> defined =[C]:-1
 ttAnchorType = "mouse"
 ttDisplayingUnit = false
 SetCraftItem = <function> defined =[C]:-1
 RightEdge = <unnamed> {
 }
 ttDisplayingAura = false
 BottomEdge = <unnamed> {
 }
 SetBackdrop = <function> defined @TipTac\ttCore.lua:1630
 FadeOut = <function> defined @TipTac\ttCore.lua:1435
 TopOverlay = <unnamed> {
 }
 GetBackdropBorderColor = <function> defined @SharedXML\Backdrop.lua:409
 GetBackdrop = <function> defined @SharedXML\Backdrop.lua:347
 SetUnitDebuff = <function> defined =[C]:-1
 SetItemByID = <function> defined =[C]:-1
 SetBuybackItem = <function> defined =[C]:-1
 SetQuestItem = <function> defined =[C]:-1
 miniMapIcon = false
 SetBorderBlendMode = <function> defined @SharedXML\Backdrop.lua:266
 SetMerchantItem = <function> defined =[C]:-1
 manualOrder = <table> {
 }
 HasBackdropInfo = <function> defined @SharedXML\Backdrop.lua:278
 ttUnit = <table> {
 }
 SetInboxItem = <function> defined =[C]:-1
 shoppingTooltips = <table> {
 }
 updateTooltip = 0.066000
 TooltipBackdropOnLoad = <function> defined @SharedXML\SharedTooltipTemplates.lua:172
 backdropInfo = <table> {
 }
 OnBackdropSizeChanged = <function> defined @SharedXML\Backdrop.lua:182
 SetUnitBuff = <function> defined =[C]:-1
 SetAction = <function> defined =[C]:-1
 SetTradePlayerItem = <function> defined =[C]:-1
 SetBackdropBorderColor = <function> defined @TipTac\ttCore.lua:1646
 SetSendMailItem = <function> defined =[C]:-1
 ttSetBackdropLocked = true
 comparing = false
 Center = <unnamed> {
 }
 ApplyBackdrop = <function> defined @TipTac\ttCore.lua:1622
 __tamedCounts = false
 ttBackdropColorApplied = false
 SetUnitAura = <function> defined =[C]:-1
 SetTradeTargetItem = <function> defined =[C]:-1
 SetQuestLogItem = <function> defined =[C]:-1
 ttSetBackdropColorLocked = true
 needsReset = true
 SetupPieceVisuals = <function> defined @SharedXML\Backdrop.lua:246
 ttAnchorPoint = "BOTTOMLEFT"
 SetQuestLogCurrency = <function> defined =[C]:-1
 BottomOverlay = <unnamed> {
 }
}

```

**Comment by frozn** [comment]
Thanks for reporting this! 👍

Fixed with new release [v22.09.17](https://github.com/frozn/TipTac/releases/tag/v22.09.17).

**Comment by NayooZ** [comment]
Thankyou!
